src/recurrence.jl

```diff
@@ -56,3 +84,12 @@ end
     return jnup1, jnu
 end
 =#
+function besselk_down_recurrence!(out, x::T, nu_range) where T
+    x2 = 2 / x
+    k = length(nu_range) - 2
+    for nu in nu_range[end-1:-1:2]
```

Comment:
`begin+1`?

Reply:
Yes that's better practice, thank you. I'm pretty sure this should work for any `AbstractRange` provided `step(nu) == 1`...